Tuto-GUDHI-cover-complex.ipynb

```diff
@@ -1062,7 +1062,7 @@
     {
```

Comment:
<p>Vincent did something in <a href="https://github.com/GUDHI/TDA-tutorial/pull/68" target="_blank">https://github.com/GUDHI/TDA-tutorial/pull/68</a> to hide these messages until <a href="https://github.com/GUDHI/gudhi-devel/issues/882" target="_blank">https://github.com/GUDHI/gudhi-devel/issues/882</a> is fixed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>